RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHE_cfi.py

```diff
@@ -163,3 +164,8 @@
 particleFlowClusterHBHEOnly = particleFlowClusterHBHE.clone(
     recHitsSource = "particleFlowRecHitHBHEOnly"
 )
+
+#--- Use DB conditions for cuts&seeds solely for Run3
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+(~phase2_common & run3_HB).toModify( particleFlowClusterHBHE,
```

Comment:
hello @abdoulline 
Do we intend to use GT based approach only for HB? Or for HB and HE both? 
The `run3_HB` is a bit confusing, I took it from your talk https://indico.cern.ch/event/1220970/contributions/5284605/attachments/2598482/4486203/HCAL_PFlowCutsSeeds_Feb24_2023.pdf , page 6; but since the talk is a bit old maybe its good to clarify the plan again. 

Thanks @RSalvatico for spotting

Reply:
Since we are discussing this, is there are reason to not do the same also for phase2 offline?  I see we're doing it in the upgrade HLT menu.

Reply:
in phase2 HLT menu, we have kept `usePFThresholdsFromDB=False`

Reply:
>  in phase2 HLT menu, we have kept usePFThresholdsFromDB=False

right, why? Is it just a matter of updating the corresponding GT?

Reply:
@swagata87  both HB and HE.  HCAL introduced "run3_HB" modifier (included in Run3 one) to emphasize that in Run3 we have only HB changed. But I do agree that general Run3 better be used. Mea culpa again, in sides I've just copy-pasted a particular config line...

Reply:
@mmusich as I've already mentioned, the initial plan was to restrict ourselves with Run3 for PDCuts. To see later if/how to extend DB conditions usage to other eras. 
NB: HCAL is not using (and is not planning to use until LS3) any DB conditions, except (quite special DB object ElectronicsMap), for Phase2.   So, extending PFCuts usage to Phase 2 may need some time/tests.  
I understand the intention to do everything at once,  but I'd suggest we come back to it later (after 13_3_X).

Reply:
> So, extending PFCuts usage to Phase 2 may need some time/tests.

isn't the same code used in Run 3 also in Phase 2? What would tests entail exactly?

Reply:
I have now changed the era name to `run3_common`. If I understand correctly this should have no effect, its just less confusing to a reader